docs/scenarios.md

```diff
@@ -509,23 +519,26 @@ which contains a green-screen in all frames intended for adversarial patch inser
   * Adversary objectives:
     * To degrade the performance of the tracker through the insertion of adversarial patches.
   * Adversary Operating Environment:
-    * Non-real time, digital and physical-like patch attacks
+    * Non-real time, physical-like patch attacks
     * Adaptive attacks will be performed on defenses.
 * Adversary Capabilities and Resources
-    * Patch size of different size/shape as dictated by the green-screen in the frames. The adversary is expected to apply a constant patch across all frames in the video.
+    * Patch size of different size/shape as dictated by the green-screen in the frames. The adversary is expected to apply a patch with constant texture across all frames in the video, but the patch relative to the sensor may change due to sensor motion.
 * **Metrics of Interest:**
   * Primary metrics:
     * mean IOU
+    * mean succss rate (mean IOUs are calculated for multiple IOU thresholds and averaged)
 * **Baseline Attacks:**
-  * [Custom Adversarial Texture with Input-Dependent Transformation](https://github.com/twosixlabs/armory/blob/r0.14.2/armory/art_experimental/attacks/carla_adversarial_texture.py)
-* **Baseline Defense**: [Video Compression](https://github.com/twosixlabs/armory/blob/r0.14.2/armory/art_experimental/defences/video_compression_normalized.py)
-* **Baseline Model Performance: (results obtained using Armory v0.14.2 and [test data](https://github.com/twosixlabs/armory/blob/r0.14.2/armory/data/adversarial/carla_video_tracking_test.py))**
+  * [Custom Adversarial Texture with Input-Dependent Transformation](https://github.com/twosixlabs/armory/blob/v0.15.1/armory/art_experimental/attacks/carla_adversarial_texture.py)
+* **Baseline Defense**: [Video Compression](https://github.com/twosixlabs/armory/blob/v0.15.1/armory/art_experimental/defences/video_compression_normalized.py)
+* **Baseline Model Performance: (results obtained using Armory v0.15.1 and [dev data](https://github.com/twosixlabs/armory/blob/v0.15.1/armory/data/adversarial/carla_video_tracking_dev.py))**
+
+| Benign Mean IoU | Benign Mean Success Rate | Adversarial Mean IoU | Adversarial Mean Success Rate | Test Size |
+|-----------------|--------------------------|----------------------|-------------------------------|-----------|
+| 0.55/0.57       | 0.57/0.60                | 0.14/0.19            | 0.15/0.20                     | 20        |
```

Comment:
I should have results for video tracking tomorrow AM

Reply:
@lcadalzo I used the default attack hyperparameters for video.

Reply:
For undefended:
```
adversarial_mean_video_tracking_mean_iou on adversarial examples w.r.t. ground truth labels: 19.88%
```

And defended:
```
adversarial_mean_video_tracking_mean_iou on adversarial examples w.r.t. ground truth labels: 22.19%
```

This differs a bit from the reported `0.14` and `0.19` respectively. I used the default `max_iter: 100` and `step_size: 0.02`

Reply:
makes sense, I actually made this change in 15f477c